pinot-core/src/main/java/com/linkedin/pinot/core/operator/blocks/InstanceResponseBlock.java

```diff
@@ -42,6 +46,9 @@ public InstanceResponseBlock(Block block) {
     try {
       _instanceResponseDataTable = intermediateResultsBlock.getDataTable();
     } catch (Exception e) {
+      _instanceResponseDataTable = new DataTableBuilder(
```

Comment:
better to check for null in the caller and throw exception. Calling code is already providing this logic.

_instanceResponseDataTable = new DataTable() // dont' need to pass schema
_instanceResponseDataTable.addException(....) 
should work